src/js/components/LogDetailsWindow/index.js

```diff
@@ -2,24 +2,24 @@
 import {useDispatch, useSelector} from "react-redux"
 import React from "react"
 
+import {Center, Left, PaneHeader, PaneTitle, Right} from "../Pane"
```

Comment:
I'm using "import-js" https://github.com/Galooshi/import-js for now. I've got an atom plugin that calls out to it. By default it groups and sorts the imports automatically. It's a little annoying when the diff is bigger just because of the sorting. We have a `.importjs.js` config file too at the root of the project.